app/lib/ca/BaseFindController.php

```diff
@@ -749,14 +749,15 @@ protected function _genExport($po_result, $ps_output_type, $ps_output_filename,
 					case '_csv':
 						$vs_delimiter = ",";
 						$vs_output_file_name = mb_substr(preg_replace("/[^A-Za-z0-9\-]+/", '_', $ps_output_filename.'_csv'), 0, 30);
-						$vs_file_extension = 'txt';
-						$vs_mimetype = "text/plain";
+						$vs_file_extension = 'csv';
+						$vs_mimetype = "text/comma-separated-values";
```

Comment:
I think I see... you've got `text/tab-separated-values` for `tsv` files, which I think is the only valid mime type.